src/components/ProjectList.vue

```diff
@@ -114,6 +121,9 @@
         .catch(function (error) {
           console.log(error)
         })
+      },
+      noProjects: function () {
```

Comment:
Probably a slightly better approach would've been the "hasProjects" way (so as to avoid the double negation in the !noProject) :) Not a big deal though.